security-manual-draft.md

```diff
@@ -0,0 +1,107 @@
+# Sicherheit in Freifunknetzen
+## Die kleine Übersicht
+Für alle die, die nur grob wissen wollen, worauf zu achten ist:
+
+### Dinge in offenen/fremden Netzwerken beachten sollte
+
+* Netzwerk als "öffentliches Netzwerk" in den Windows-Firewall Einstellungen markieren
+* Freigaben von Daten ins Netzwerk abschalten
+* HTTPS für alle Seiten, wo man sich anmeldet
+* SSL/TLS in E-Mail Programmen aktivieren
+* Sicherheitswarnungen im Browser beherzigen
+* (optional) Verwenden von VPN
+
+### Dinge die man in offenen/fremden Netzwerken vermeiden sollte
+
+* Filesharing
+* Öffnen von Ports
+* "Firmen-/Heimnetzwerk"-Einstellungen in den Windows-Firewall Einstellungen nutzen
+
+
+## WLAN Sicherheit
+### Was bedeutet das überhaupt?
+Bei der "WLAN-Sicherheit" bzw. richtiger der WLAN-Verschlüsselung handelt es 
+sich ausschließlich um die Verschlüsselung, der durch die Luft übertragenen 
+Daten, zwischen dem Client (Smartphone, Tablet, Laptop, etc.) und dem 
+Accesspoint ("WLAN-Router", Freifunkknoten, etc.). Hierfür stehen verschiedene 
+Mechanismen zur Verfügung, die hier im Anschluss grob erklärt werden sollen.
+
+### WPA/PSK und WPA2/PSK
+Bei WPA/PSK und WPA2/PSK handelt es sich um die gängigsten 
+Verschlüsselungsmethoden in Deutschland. Sie sind inzwischen als Nachfolger von 
+WEP die Hauptverschlüsselungsmechanismen für private WLANs. WPA beschreibt 
+hierbei, wie die Nachrichten verschlüsselt werden, aber der weit wichtigere 
+Teil ist das Anhängsel PSK. PSK steht für Pre Shared Key. Der Schlüssel muss 
+also sowohl dem Accesspoint als auch dem Benutzer bekannt sein. Das Problem 
+hierbei ist, dass **jeder** Benutzer des Netzwerks **diesen für alle gleichen** 
+Schlüssel kennen muss, wodurch auch jeder, der den Schlüssel besitzt den 
+gesamten WLAN Verkehr mitlesen kann.
+
+### WPA/EAP und WPA2/EAP
+Hierbei handelt es sich um einen Standard, wie er in Firmen häufig verwendet 
+wird. Die Verschlüsselungsmethode ist die Gleiche, wie bei privaten WLANs auch. 
+Den Unterschied macht hier das EAP, welches für "Extensible Authentication 
+Protocol" steht und verschiedene Mechanismen bietet die Verschlüsselung zu 
+nutzen. Gängig ist hier die Anmeldung an einem WLAN, wie man es auch sonst 
+gewohnt ist, über einen Usernamen und ein Passwort, aber auch ein Zugriff über 
+ein Zertifikat ist möglich und hängt von der Konfiguration ab. 
+ 
+### Keine (offen/unverschlüsselt)
+Nun ja, wie der Name schon sagt, findet hier einfach keine Verschlüsselung 
+statt, der Zugriff ist also für jeden möglich und der Datenverkehr sichtbar.
```

Comment:
Ja, könnte man so entschäften. Wobei ich eigentlich davon ausgehe, dass das was darunter steht die Sache erklärt. Mal schauen, vielleicht finde ich eine Treffendere Formulierung. Mit Beispielen würde ich in dem Fall aber mal eher ungern arbeiten.